src/storage/engine/rocksdb_engine.rs

```diff
@@ -249,6 +250,14 @@ impl Engine for RocksEngine {
     }
 
     fn async_snapshot(&self, _: &Context, cb: Callback<Self::Snap>) -> Result<()> {
+        fail_point!("rockskv_async_snapshot", |_| Err(box_err!(
```

Comment:
So many failpoints.

Reply:
We have one for raftkv async snapshot. So let's add one for rocksdb kv as well. In coprocessor tests only rocksdb engine is used.